pages/blog/index.js

```diff
@@ -29,14 +33,20 @@ function BlogIndex({ articles }) {
     <>
       <Head title={pageTitle} />
       <HeroBanner title={pageTitle} />
-      <ul>
+
+      <ul className={styles.blogsList}>
         {articles.map(article => {
           return (
-            <li key={article}>
-              <Link href="/blog/[article]" as={`/blog/${article}`}>
-                <a>{article}</a>
-              </Link>
-            </li>
+            <ImageCard
+              alt="Two women pair programming"
+              imageSource={`${s3}stock_paired-programming.jpg`}
+            >
+              <li key={article}>
```

Comment:
Instead, I'd wrap the `ImagCard` inside `li`.

Reply:
I will make this change.

Reply:
Thank you for reviewing my PR. I will make this change.